packages/xod-arduino/platform/runtime.cpp

```diff
@@ -153,6 +153,7 @@ void setup() {
     delay(100);
 #ifdef XOD_DEBUG
```

Comment:
I think it's better to use the same `XOD_DEBUG_SERIAL` here and call `begin` and `setTimeout` for the simulation too.

P.S.
Now `begin` in Simulation Serial is just a noop method and there is no `setTimeout` method. But it's a detail of implementation, which we should not think about in this context.